apis/helm.integrations.flux.weave.works/v1alpha/types.go

```diff
@@ -21,15 +21,15 @@ type FluxHelmRelease struct {
 // FluxHelmReleaseSpec is the spec for a FluxHelmRelease resource
 // FluxHelmReleaseSpec
 type FluxHelmReleaseSpec struct {
-	ChartGitPath string         `json:"chartGitPath"`
-	ReleaseName  string         `json:"releaseName,omitempty"`
-	Values       FluxHelmValues `json:"values,omitempty"`
+	ChartGitPath   string `json:"chartGitPath"`
+	ReleaseName    string `json:"releaseName,omitempty"`
+	FluxHelmValues `json:",inline"`
```

Comment:
This indicates that FluxHelmValues will slurp whatever fields are not chartGitPath or releaseName within the FluxHelmReleaseSpec part. However as FluxHelmValues struct has a field defined as `json:"values,omitempty"`, it will only pick "values". Is that correct? Has it been empirically confirmed that an additional field, different to chartGitPath, releaseName and values will be discarded? (I think it should be given the definition.)

Reply:
Here's an example showing multiple `json:",inline"`s and additional values: https://play.golang.org/p/T1D3YikyKQ2

I believe the schema that is used to unmarshal the string takes into account the inlines at that point. Additional values would be discarded if not defined in the struct as usual.

Reply:
Thank you :+1: